browser/layout/partials/ctm.html

```diff
@@ -2,20 +2,22 @@
 var classes = document.documentElement.className; {{!note: once we ditch support for ie9 we can just use classlist}}
 classes = classes.replace(/\bno-js\b/, 'js'); {{!note: needed for JS that's run out of the scope of the bootstrap (inline onclicks, etc)}}
 
-var getRandomValuesAvailable;
-var cryptoApi = window.crypto || window.msCrypto;
-if (cryptoApi) {
-	getRandomValuesAvailable = 'getRandomValues' in cryptoApi;
+var supportsBlobConstructor;
+try {
+	new Blob;
```

Comment:
This looks like it also knocks out UCBrowser (10% global usage???).  Is that intentional?